README.md

```diff
@@ -150,6 +129,79 @@ If CallKit has been set up correctly, Tom's device will ring like a phone call,
 
 Note: Due to an issue with ARKit, you cannot start a Remote Streem from a view using the camera https://forums.developer.apple.com/message/411888#411888
 
+#### Roles
+
+Roles dictate which side of the streem your on as in whether you're sharing your video or receiving video. They also affect which tools are available to you. The different roles are:
```

Comment:
`Roles dictate which side of the streem you're on: whether you are providing or receiving the video feed.`

Reply:
👍